Math/Vector2.js

```diff
@@ -81,6 +81,14 @@ PIXI.Point.prototype.Dist = function(other) {
 }
 PIXI.ObservablePoint.prototype.Dist = PIXI.Point.prototype.Dist
 
+/* Multiplies this vector by a scalar */
```

Comment:
Yeah, but from what I could tell, it wants an outpoint and does not do the multiplication in place. Creating another Point() object feels pointless to me. It would also take an extra line of code to make the outPoint. (Maybe we could pass the point directly there, but making a new function felt better to me) This is more a stylistic choice than anything else,